### PR TITLE
Readme linux libgdal dev

### PR DESCRIPTION
**Pull request by Tonow**
Readme :
 * Add requirement for linux libdgdal-dev
 * Add Link repo requirement.txt
 * Add Nav menu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,26 +12,65 @@ A Python wrapper around the [OSRM API](https://github.com/Project-OSRM/osrm-back
...
 - GDAL
 
+
+## Sum up command to install
+
+### Ubuntu
+
+```
+sudo apt-get install libgdal-dev \
+&& curl -o requirements.txt https://raw.githubusercontent.com/ustroetz/python-osrm/master/requirements.txt \
```
Comment (zifeo) [inline README.md]: I would prefer not to encourage bypassing the pipit requirements as a new version should be soon pushed to fix the current issue. @ustroetz

File: README.md [hunk 2]
```diff
@@ -12,26 +12,65 @@ A Python wrapper around the [OSRM API](https://github.com/Project-OSRM/osrm-back
...
+&& pip install -r requirements.txt \
+&& pip install osrm
+```
+
+## Running the test suite
```
Comment (zifeo) [inline README.md]: Let's make it a third level title instead.

**Comment by Tonow** [comment]
@zifeo you think it's better ?

**Comment by zifeo** [comment]
@Tonow thanks, I was not aware you updated the PR. @ustroetz WDYT?

**Comment by ustroetz** [comment]
Thanks! Again, sorry for the slow response. I was on a longer leave.

**Pull request merged**